src/internal/connector/onedrive/collections_test.go

```diff
@@ -1344,6 +1346,42 @@ func (suite *OneDriveCollectionsSuite) TestGet() {
 			expectedFolderPaths: nil,
 			expectedDelList:     nil,
 		},
+		{
+			name:   "OneDrive_OneItemPage_DeltaError",
+			drives: []models.Driveable{drive1},
+			items: map[string][]deltaPagerResult{
+				driveID1: {
+					{
+						err: getDeltaError(),
+					},
+					{
+						items: []models.DriveItemable{
+							driveItem("file", "file", testBaseDrivePath, true, false, false),
```

Comment:
can you add another item that is in a different folder please? DoNotMerge should be set for all collections in the drive

Can you also add another drive with a non-expired token? DoNotMerge should only be set for the collections in the drive that expired